.github/workflows/publish_navigation_component_ado.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: publish-navigation-component-ado
+# This workflow builds and publishes in ADO the Foldable Navigation Component artifacts (all of them).
+
+on:
+  push:
+    tags:
+      - 'dualscreen*'
```

Comment:
Please change it for 'navigation*' since we will use that key-word to trigger this workflow

Reply:
Done